Meta/CMake/Findlibjxl.cmake

```diff
@@ -0,0 +1,32 @@
+include(FindPackageHandleStandardArgs)
```

Comment:
If this file is lifted directly from SDL_image, we need to include the text of SDL's zlib license at the top of the file.

Reply:
Added the license